examples/wcf/server-aspnetframework/Examples.Wcf.Server.AspNetFramework.csproj

```diff
@@ -0,0 +1,74 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <PropertyGroup>
+    <TargetFramework>net48</TargetFramework>
```

Comment:
Is 4.8 a requirement? Just wondering if we should do 4.6.2 since that is generally what we target for .NET Framework (lowest supported version).

Reply:
Nope, not a requirement....updated to `net462`